packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -45,6 +55,8 @@ export type NotifiSubscriptionData = Readonly<{
   setConversationId: (conversationId: string) => void;
   userId: string;
   setUserId: (userId: string) => void;
+  demoPreview: DemoPreview | null;
+  setDemoPreview: React.Dispatch<React.SetStateAction<DemoPreview | null>>;
```

Comment:
I think setting this via prop solely via prop feels cleanest

Reply:
Already Migrated demoPreview to newly created 'NotifiDemoPreviewContext' that handles demoPreview related context.So this section no longer exists.